assets/app/scripts/controllers/create.js

```diff
@@ -94,6 +107,213 @@ angular.module('openshiftConsole')
       $scope.filter.tag = tag;
     };
 
+    $scope.aceLoaded = function(editor) {
+      $scope.editor.session = editor.getSession();
+      $scope.editor.session.setOption('tabSize', 2);
+      $scope.editor.session.setOption('useSoftTabs', true);
```

Comment:
Since this isn't used in the view anywhere, let's make it a local var to the controller.

```javascript
var editorSession;
$scope.aceLoaded = function(editor) {
      editorSession = editor.getSession();

...
```